src/actions/channels.js

```diff
@@ -512,25 +512,11 @@ export function fetchMyChannelsAndMembers(teamId: string): ActionFunc {
             },
         ]), getState);
 
-        const channelsRequest = Client4.getMyChannels(teamId);
-        const channelMembersRequest = Client4.getMyChannelMembers(teamId);
-
         let channels;
-        try {
-            channels = await channelsRequest;
-        } catch (error) {
-            forceLogoutIfNecessary(error, dispatch, getState);
-            dispatch(batchActions([
-                {type: ChannelTypes.CHANNELS_FAILURE, error},
-                {type: ChannelTypes.CHANNEL_MEMBERS_FAILURE, error},
-                logError(error),
-            ]), getState);
-            return {error};
-        }
-
         let channelMembers;
         try {
-            channelMembers = await channelMembersRequest;
+            channels = await Client4.getMyChannels(teamId);
+            channelMembers = await Client4.getMyChannelMembers(teamId);
```

Comment:
I think you need to do:

```javascript
const channelRequest = Client4.getMyChannels(teamId);
const memberRequest = Client4.getMyChannelMembers(teamId);
await channelRequest;
await memberRequest
```
So that they happen in parallel

Reply:
@jwilander @hmhealey changed so the requests are executed in parallel